### PR TITLE
Parse AnalysisRuns to improve UI

### PR DESCRIPTION
**Pull request by mvgmb**
Parse AnalysisRun to take advantage of existing UI elements.

Before:
Screen Shot 2022-06-01 at 8 41 25 PM

After:
Screen Shot 2022-06-01 at 8 42 50 PM

### CODE REVIEW

File: ui/src/index.tsx [hunk 1]
```diff
@@ -110,7 +111,36 @@ const parsePodStatus = (pod: any) => {
...
+        name: node.name,
+        namespace: node.namespace,
+        resourceVersion: node.version,
+        uid: node.uid,
+      },
+      revision: parseRevision(node),
+      status: parseAnalysisRunStatus(node.health.status),
+    }) as RolloutAnalysisRunInfo);
+
+const parseAnalysisRunStatus = (status: string): string => {
```
Comment (mvgmb) [inline ui/src/index.tsx:130]: `parseAnalysisRunStatus` function converts Argo CD status to Argo Rollouts' Analysis Run status.

File: ui/src/index.tsx [hunk 2]
```diff
@@ -110,7 +111,36 @@ const parsePodStatus = (pod: any) => {
...
+    case 'Healthy':
+      return 'Successful';
+    case 'Progressing':
+      return 'Running';
+    case 'Degraded':
+      return 'Failure';
+    default:
+      return 'Error';
+  }
+}
```
Comment (mvgmb) [inline ui/src/index.tsx:141]: `parseAnalysisRunStatus` function converts Argo CD status to Argo Rollouts' Analysis Run status.

**Review by httin — APPROVED** [review]
this is nice. Can this enhancement go to the next release, please?

**Comment by alexef** [comment]
How does this relates with: https://github.com/argoproj/argo-rollouts/pull/1910 ? as 1.3.0 just got released, I don't feel I understand the exact relationship between that and this extension.

Does the extension get the changes automatically? Or is the extension a completely separate UI that hooks into CD?

**Comment by alexef** [comment]
Answered myself, the newly introduced view will be fed with data from argocd.

**Review by alexef — APPROVED** [review]

**Review by zachaller — APPROVED** [review]

**Review by leoluz — APPROVED** [review]

**Pull request merged**